control/config.py

```diff
@@ -20,7 +20,40 @@
     'control.squeeze_time_response': None,
     'forced_response.return_x': False,
 }
-defaults = dict(_control_defaults)
+
+
+class DefaultDict(dict):
```

Comment:
Thanks but if I  understand correctly, that 10 yrs old question and answer does not really reflect the current Python 3 reality anymore: https://docs.python.org/3/library/collections.html#collections.UserDict

Reply:
It might be an implementation detail and not guaranteed for the future, but subclassing `collections.UserDict` also works for `.get()` `.copy()` and `.update()` automatically as opposed to subclassing plain `dict()`. Thanks for the hint!